scripts/agent_bridge_provider.py

```
@@ -0,0 +1,92 @@
+class Provider:
+    def plan(self, request):
+        raise NotImplementedError
+
+
+class FakeProvider(Provider):
+    def plan(self, request):
+        return fake_llm_bridge.plan_for_request(request)
+
+
+class OpenAICompatibleProvider(Provider):
+    def __init__(self, api_key, model, base_url, timeout=10.0):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)